contracts/sentinel/guards/examples/ReEntrancyGuard.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Example contracts should be moved to the mocks folder

Reply:
This isn't a mock, it need to be here so that people see it as an example.